### PR TITLE
Rereading

### PR DESCRIPTION
**Pull request by yassine-ouaamou**
As I was rereading the documentation, here are some fixes I made.
Can you please check specialy the yaml conf parts of authentication.md to see if I haven't messed up the double quotes? (like this: **- PROVIDERS__SAML__CONFIG__LABEL="Login with SAML"**)

### CODE REVIEW

**Comment by yassine-ouaamou** [comment]
@SouadHadjiat What you think about the added content of configuration.md?
Thanks!

File: docs/deployment/configuration.md [hunk 1]
```diff
@@ -178,46 +178,49 @@ Here are the configuration keys, for both containers (environment variables) and
...
+| playbook_manager:lock_key           | PLAYBOOK_MANAGER__LOCK_KEY                       | publisher_manager_lock     | Lock key for the manager in Redis                   |
+| playbook_manager:interval           | PLAYBOOK_MANAGER__INTERVAL                       | 60000                      | Interval to check new playbooks                     |
+| -                                   | -                                                | -                          | -                                                   |
+| file_index_manager:enabled          | FILE_INDEX_MANAGER__ENABLED                      | `true`                     | Enable/disable the file indexing manager            |
+| file_index_manager:lock_key         | FILE_INDEX_MANAGER__LOCK_KEY                     | file_index_manager_lock    | Lock key for the manager in Redis                   |
+| file_index_manager:interval         | FILE_INDEX_MANAGER__INTERVAL                     | 6000                       | Interval to check new files                         |
```
Comment (SouadHadjiat) [inline docs/deployment/configuration.md:223]: There is a configuration missing : 
`"stream_lock_key": "file_index_manager_stream_lock"` => file_index_manager:stream_lock_key

Default interval is going to change, shouldn't be 6000 (@richard-julien)

File: docs/deployment/configuration.md [hunk 2]
```diff
@@ -178,46 +178,49 @@ Here are the configuration keys, for both containers (environment variables) and
...
+| retention_manager:enabled           | RETENTION_MANAGER__ENABLED                       | `true`                     | Enable/disable the retention manager                |
+| retention_manager:lock_key          | RETENTION_MANAGER__LOCK_KEY                      | retention_manager_lock     | Lock key for the manager in Redis                   |
+| retention_manager:interval          | RETENTION_MANAGER__INTERVAL                      | 60000                      | Interval to check items to be deleted (in ms)       |
+| -                                   | -                                                | -                          | -                                                   |
+| notification_manager:enabled        | NOTIFICATION_MANAGER__ENABLED                    | `true`                     | Enable/disable the notification manager             |
+| notification_manager:lock_key       | NOTIFICATION_MANAGER__LOCK_KEY                   | notification_manager_lock  | Lock key for the manager in Redis                   |
```
Comment (SouadHadjiat) [inline docs/deployment/configuration.md:206]: this is not up to date, we should have two lock keys "lock_live_key" and "lock_digest_key" : 

```
"notification_manager": {
    "enabled": true,
    "lock_live_key": "notification_live_manager_lock",
    "lock_digest_key": "notification_digest_manager_lock",
    "interval": 10000
  },
``

**Comment by SouadHadjiat** [comment]
In the configuration, I think that there is also these managers configuration missing : 

```
"activity_manager": {
    "enabled": true,
    "lock_key": "activity_manager_lock"
  },
  "connector_manager": {
    "enabled": true,
    "lock_key": "connector_manager_lock",
    "works_day_range": 7,
    "interval": 60000
  },
  "import_csv_built_in_connector": {
    "enabled": true,
    "interval": 10000,
    "validate_before_import": false
  },
```

File: docs/administration/audit/overview.md
Comment (NiQuintin) [inline docs/administration/audit/overview.md:1]: @@ -50,7 +50,7 @@ Audit will produces **console/logs** files along with user interface elements.

No 's' on verbs conjugated in the third person singular in the future tense.

File: docs/deployment/configuration.md (hunk 1 above)
Comment (yassine-ouaamou) [reply docs/deployment/configuration.md]: Put to 60000 as we said!

**Pull request merged**